packages/react-component-library/package.json

```diff
@@ -28,8 +28,8 @@
   "scripts": {
     "analyze": "webpack --mode production --config=webpack/analyze.js",
     "build": "run-s build:es build:cjs types",
-    "build:cjs": "NODE_ENV=cjs webpack --mode production --config=webpack/prod.js",
-    "build:es": "NODE_ENV=es babel src --out-dir \"dist/es\" --extensions \".ts,.tsx\"",
+    "build:cjs": "BABEL_ENV=cjs webpack --mode production --config=webpack/prod.js",
+    "build:es": "BABEL_ENV=es babel src --out-dir \"dist/es\" --extensions \".ts,.tsx\"",
     "build:dev": "RNDS_LOG_LEVEL=debug run-s build:ts:dev types",
```

Comment:
The dev commands look like they could do with a bit of love – but it's not directly related to updating the loaders or Storybook to use Webpack 5 so I've left it out of this PR.

Reply:
I thought similar - perhaps worth adding a ticket

Reply:
Created #2805